applications/vanilla/models/class.discussionmodel.php

```diff
@@ -654,14 +631,6 @@ public function calculate(&$Discussion) {
         $Discussion->CategoryUrlCode = $Category['UrlCode'];
         $Discussion->PermissionCategoryID = $Category['PermissionCategoryID'];
 
-        // Add some legacy calculated columns.
-        if (!property_exists($Discussion, 'FirstUserID')) {
-            $Discussion->FirstUserID = $Discussion->InsertUserID;
-            $Discussion->FirstDate = $Discussion->DateInserted;
-            $Discussion->LastUserID = $Discussion->LastCommentUserID;
-            $Discussion->LastDate = $Discussion->DateLastComment;
-        }
-
```

Comment:
I think leaving something like this in for a period of time would help with the deprecation process.